src/commands/course/subcommands/switchSection.ts

```diff
@@ -0,0 +1,66 @@
+import * as Discord from '../../../Discord';
+import SubCommand from '../../../SubCommand';
+import CommandOption from '../../../CommandOption';
+import { boldCourse } from '../_common';
+
+const switchSection = new SubCommand({
+  name: 'switch-section',
+  displayName: 'Switch Course Section',
+  description: "Switch section of a course you're in",
+  options: [
+    new CommandOption({
+      name: 'role',
+      description: 'The course role',
+      required: true,
+      type: Discord.CommandOptionType.ROLE,
+    }),
+    new CommandOption({
+      name: 'section-number',
+      description: 'The section to switch to',
+      required: true,
+      type: Discord.CommandOptionType.INTEGER,
+    }),
+  ],
+  handler: async ctx => {
+    const guildId = ctx.mustGetGuildId();
+    const roleId = ctx.getArgument<string>('role') as string;
+    const sectionNum = parseInt(
+      ctx.getArgument<string>('section-number') as string,
+    );
```

Comment:
1. Should there be a regex validation here like used for course numbers? For example, returning an error message if the section number provided is not a number or if too few/many digits.
2. Also, is there any standardization with leading zeros? Sections `1` and `01` should be saved as the same.

Reply:
It's `Discord.CommandOptionType.INTEGER`, so it's guaranteed to be an integer. And parseInt ignores leading zeros.